### ISSUE TITLE
Module not found: Error: Default condition should be last one

### ISSUE THREAD

**Issue by Prudhvi0717** [pocka/slack-message-parser#36]
Hey There,

I have tried to import slack message parser in my react js app but I got this error,

> Module not found: Error: Default condition should be last one

After a big struggle, I fixed it by modifying the exports property in package.json.
Before modification : 
```
  "exports": {
    ".": {
      "import": {
        
        "default": "./esm/mod.js",
        "types": "./types/mod.d.ts"
        
      },
      "require": {
        "default": "./script/mod.js",
        "types": "./types/mod.d.ts"
      }
    }
  }
```

**After modification, placed the default condition at last in the import field like this :**

```
  "exports": {
    ".": {
      "import": {
        
        "types": "./types/mod.d.ts",
        "default": "./esm/mod.js"
      },
      "require": {
        "default": "./script/mod.js",
        "types": "./types/mod.d.ts"
      }
    }
  }
```

This fixed the error

**Comment by pocka** [comment]
Thanks for reporting, investigation and also a fix!

This is caused by the old version of `dnt` module (Deno module to convert Deno package to Node's one) and has been fixed in the later version. I confirmed that upgrading the module fix the incorrect order on my local machine. Will publish a new version after evaluating dnt's changelogs.

https://github.com/denoland/dnt/issues/228

**Comment by pocka** [comment]
Fixed in v3.0.1.
